lib/slack/util.py

```diff
@@ -86,7 +91,8 @@ def get_year_month_day(day_content):
         Returns:
             str:year, str:month, str:day
     """
-    year, month, day = str(day_content["auxiliary_data"]["year"]), str(day_content["auxiliary_data"]["month"]), str(day_content["auxiliary_data"]["day"])
+    year, month, day = str(day_content["auxiliary_data"]["year"]), str(day_content["auxiliary_data"]["month"]), str(
+        day_content["auxiliary_data"]["day"])
```

Comment:
Two more tabs would align the right side of the assignment statement.

Reply:
done